### PR TITLE
Feature/db

### PR DESCRIPTION
**Pull request by efox3590**
Updated the README. Let me know if you want to change/add anything.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,26 +22,27 @@ $ npm start
...
 * ID
 * email
 * first_name
 * last_name
 * profile_pic
 
-### follower
+### followers
 * user_id
 * follower_id
```
Comment (chickenricer) [inline README.md]: should be followed_id

Comment (efox3590) [inline README.md]: It should be changed now

File: README.md [hunk 2]
```diff
@@ -19,6 +19,23 @@ $ npm install
...
+* PassportJS
+* Firebase—for image hosting
+
+# Uses
+* User can create an account
+* Post image with caption
+* See all image posts 
+* Edit post
+* Delete post
+* Comment on posts
```
Comment (chickenricer) [inline README.md]: this feature has not been built out yet.

File: README.md [hunk 3]
```diff
@@ -19,6 +19,23 @@ $ npm install
 ```
 $ npm start
 ```
+# Web Dev Tools
+* Node.js
+* Express
+* PostgreSQL
+* Sequelize
```
Comment (chickenricer) [inline README.md]: we didn't use this in our project.

File: README.md [hunk 4]
```diff
@@ -19,6 +19,23 @@ $ npm install
...
+* PassportJS
+* Firebase—for image hosting
+
+# Uses
+* User can create an account
+* Post image with caption
+* See all image posts
```
Comment (chickenricer) [inline README.md]: at the moment, i have the user's feed show all other followed users activity. own activity is displayed only the user's profile page. do you think i should change it so the user's photos also appear in the feed page?

Comment (efox3590) [inline README.md]: I don't think it really makes a difference. I meant you can see all your posts in your profile view.

**Pull request merged**